src/Core/src/Hosting/ImageSources/IImageSourceServiceProvider.cs

```diff
@@ -10,8 +10,10 @@ public interface IImageSourceServiceProvider : IServiceProvider
 
 		IImageSourceService? GetImageSourceService(Type imageSource);
 
+		[Obsolete("Use GetImageSourceService instead.")]
 		Type GetImageSourceServiceType(Type imageSource);
 
+		[Obsolete("Use GetImageSourceService instead.")]
 		Type GetImageSourceType(Type imageSource);
```

Comment:
For these, add a default interface method to throw. And we can remove in .NET 10.